.chloggen/loki-exporter-add-raw-log-export-format.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: translator/loki
+note: Loki add raw log export format
+issues: [ 18888 ]
```

Comment:
Rebasing issue?

Reply:
#19772 should be merged first, sorry. I rebased on top of it to add tests and a few more things to ignore.